### PR TITLE
DOCS: tidy up wording, spelling.

### PR DESCRIPTION
**Pull request by SpiritLevel**

### CODE REVIEW

File: docs/en/02_Developer_Guides/16_Execution_Pipeline/index.md [hunk 1]
```diff
@@ -54,13 +51,13 @@ If no file can be directly matched, control is handed off to `framework/main.php
 		RewriteRule silverstripe-cache(/|$) - [F,L,NC]
 		RewriteRule composer\.(json|lock) - [F,L,NC]
 
-		# Process through SilverStripe if no file with the requested name exists.
-		# Pass through the original path as a query parameter, and retain the existing parameters.
+		# If no file with the requested name exists, pass the request over to Silverstripe's main.php script,
```
Comment (willmorgan) [inline docs/en/02_Developer_Guides/16_Execution_Pipeline/index.md]: SilverStripe, not Silverstripe

File: docs/en/02_Developer_Guides/16_Execution_Pipeline/index.md [hunk 2]
```diff
@@ -72,57 +69,54 @@ SilverStripe can also operate without this level of rewriting, in which case all
...
+ * The `Controller` can optionally hand off control to additional, nested controllers.
+ * The `Controller` optionally renders a response body through `SSViewer` [templates](../templates).
  * The `[api:RequestProcessor]` is called to post-process the request to allow 
-further filtering before content is sent to the end user
- * The response is output to the client
+further filtering before content is sent to the end user.
+ * The response is output to the client.
 
-## Request Preprocessing and Postprocessing
+## Request Pre-processing and Post-processing
```
Comment (willmorgan) [inline docs/en/02_Developer_Guides/16_Execution_Pipeline/index.md]: pre-processing and post-processing

**Comment by dhensby** [comment]
:(

**Comment by SpiritLevel** [comment]
No need to frown! I still have the edits. I am experimenting with the installer and messed my repo up. I'll submit a PR when I get this sorted.

**Comment by willmorgan** [comment]
@SpiritLevel if you have any issues/questions just drop me a message

**Comment by SpiritLevel** [comment]
Thanks @willmorgan . Can you make time pass more slowly ???? :P

**Pull request closed without merging**